### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sam951013**
I changed some words.
我改了一些翻譯錯誤。

### CODE REVIEW

**Comment by sam951013** [comment]
pull request is mean 拉取請求 not 提取要求.

**Comment by sam951013** [comment]
#sign-off

**Comment by crsuzukimsft** [comment]
Thank you. We are reviewing the request for updating the translation for pull request globally in this repo. For the time being this change request needs to be put on hold.

File: i18n/vscode-language-pack-zh-hant/README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@
 ## 參與貢獻
 
 若要對翻譯提出改善意見反應，請在 [vscode-loc](https://github.com/microsoft/vscode-loc) 存放庫中建立問題。
-翻譯字串會在 Microsoft 當地語系化平台中維護。只有在 Microsoft 當地語系化平台中才能進行變更，並匯出至 vscode-loc 存放庫。因此，vscode-loc 存放庫中不接受提取要求。
+翻譯字串會在 Microsoft 當地語系化平台中維護。只有在 Microsoft 當地語系化平台中才能進行變更，並匯出至 vscode-loc 存放庫。因此，vscode-loc 存放庫中不接受拉取請求。
```
Comment (huajiankan) [inline i18n/vscode-language-pack-zh-hant/README.md:15]: test

**Review by TylerLeonhardt — APPROVED** [review]

**Comment by TylerLeonhardt** [comment]
Thanks for fixing this!

**Pull request merged**